package.json

```diff
@@ -34,6 +37,7 @@
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^13.1.5",
     "@types/jest": "^26.0.23",
+    "@types/lodash.get": "^4.4.6",
```

Comment:
Don't think we need this as a dev dependency since it's already above.

Can also update the above to the latest version.